Modules/Segmentation/LevelSetsv4/include/itkUpdateWhitakerSparseLevelSet.hxx

```diff
@@ -860,8 +853,7 @@ UpdateWhitakerSparseLevelSet<VDimension, TLevelSetValueType, TEquationContainer>
 {
   ZeroFluxNeumannBoundaryCondition<LabelImageType> spNBC;
 
-  typename NeighborhoodIteratorType::RadiusType radius;
-  radius.Fill(1);
+  auto radius = MakeFilled<typename NeighborhoodIteratorType::RadiusType>(1);
```

Comment:
Should these now be "const auto" ?  

I'm approving, and think this PR should not be held up to add const, but it might be another benefit of using MakeFilled.

Reply:
> Should these now be "const auto" ?

Good question @hjmjohnson Actually maybe even "constexpr auto"!!! However, it's a bit risky to just blindly add const or constexpr to all such cases, as those variables might need to be modified later on.  So I'm hoping that this can be achieved my a follow-up PR, using some intelligent code analysis tool, which checks that those variables aren't modified after their initialization 😃  

Thanks for your approval!